src/Contacts/tests/ContactDetectors/FixedFootDetectorUnitTest.cpp

```diff
@@ -0,0 +1,168 @@
+/**
+ * @file FixedFootDetectorUnitTest.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <catch2/catch.hpp>
+
+#include <BipedalLocomotion/Contacts/ContactPhaseList.h>
+
+#include <BipedalLocomotion/ContactDetectors/FixedFootDetector.h>
+#include <BipedalLocomotion/ParametersHandler/IParametersHandler.h>
+#include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
+
+using namespace BipedalLocomotion::ParametersHandler;
+using namespace BipedalLocomotion::Contacts;
+
+struct FixedFootState
+{
+    EstimatedContact leftFoot;
+    EstimatedContact rightFoot;
```

Comment:
Should this be a `PlannedContact` instead of an `EstimatedContact`?

Reply:
Nevermind, this is on the output end of the planner and input end of the estimator... so, `EstimatedContact` is fitting.